content/how-to/rackspace-fabric/fabric-ticketing/index.md

```diff
@@ -64,7 +64,23 @@ The user must be granted ticketing permissions for the ticket navigation items t
 <img width="257" alt="FT11" src="https://user-images.githubusercontent.com/98875667/156183533-ce5b707d-2d71-45ea-b626-2c5c1ec7c9e4.png">
 
 4. Filter: This filter option can filter your tickets using options such as Device, Created By, Accounts, and Date Range 
-5. Refresh List: When you click on Refresh List option, the most recently created ticket appears on the top of the ticket list. 
+5. Refresh List: When you click on Refresh List option, the most recently created ticket appears on the top of the ticket list.
+
+##### Export Tickets
+<img class="fig-img" src="/support/how-to/fabric-ticketing/export-tickets_01.png" alt="An expanded drop-down showing the export ticket and close ticket options">
+
+**NOTE**: Make sure ticket list item checkboxes are not selected. The **Export Tickets** feature does not support exporting individual tickets based on the ticket item checkboxes. The option will be disabled if any checkboxes are selected.
+
+To **Export Tickets** do the following:
+1. Select **Actions** > **Export Tickets**.
+
+<img class="fig-img" src="/support/how-to/fabric-ticketing/export-tickets_02.png" alt="Export ticket popover showing radio button options to export Open Tickets, Closed Tickets, or Pending Customer tickets">
```

Comment:
Change tickets to Tickets

Reply:
@asmita32675 `Open Tickets` and `Closed Tickets` are the actual titles of the options. The third option is titled `Pending Customer`, so I left the following `tickets` word lowercase. Do you still feel I need to capitalize `tickets`?